lessons/pydata/pandas_core/index.ipynb

```diff
@@ -1769,6 +1769,13 @@
     "**Úkol:** Co se stane, když totéž učiníš se `Series`?"
```

Comment:
<p>:-D Možná bych jen drobně upřesnil 1, 2, 6</p><p> </p><ol><li><code>2 * len(pokemoni)</code>?</li><li> Dát tam rovnou konkrétní jména pokémonů?</li><li>:+1:</li><li>:+1:</li><li>:+1:</li><li>Tu podmínku prozkoumat manuálně, je to tak?</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='108'/>

Reply:
<ol><li>Áno, alebo možeš volať df.shape, df.size, alebo si ho zobraziť a manuálne sa na spodku pozrieť koľko má rows a to zdvojnásobiť. Možností tu je veľa.</li><li>Skôr som tam smeroval k tomu využiť df.columns, upravím text tak, aby to bolo jasnejšie.</li></ol><p>Šestku ako píšeš, sú len 3 Pokemoni na "i".</p><p> </p>